server/manager/src/main/java/org/apache/accumulo/manager/tableOps/delete/PreDeleteTable.java

```diff
@@ -33,6 +33,11 @@
 
 public class PreDeleteTable extends ManagerRepo {
 
+  public static String createDeleteMarkerPath(String instanceId, String tableId) {
```

Comment:
Would it be better to use TableId instead of String to strongly type the parameter?  Likely more of a personal preference.

Along the same lines - does using canonical instead of toString provide a benefit?  To me, canonical has a stronger connotation - but current implementation has same effect.  Probably should stick with what would be more common in the code - canonical appears 289 times in the code according to my IDE find usages.